backend/src/Flow.package/Thing.class/instance/removeThing..st

```diff
@@ -2,6 +2,7 @@ actions
 removeThing: aThing
 	"Removes aThing from the things that belong to this thing"
 	
+	self halt.
```

Comment:
forgotten halt. 
@fvozzi I hope is was't there for anything serious. If you remember anything suspicious about this please shout!

I'm removing it.